docs/smart_scripts.md

```diff
@@ -4174,9 +4174,8 @@ This is the probability of the event to occur as a percentage from 0-100. So, if
 </p></td>
 <td><p><br />
 </p></td>
-<td><p><br />
-</p></td>
-<td><p>All players up to maxCount within the specified range.</p></td>
+<td><p>>0 it will try all possible targets in max instead of min-maxing.</p></td>
```

Comment:
You mean, 
When target.o is set and it didnt find any targets in min-max range, it will try all possible targets in max range instead.